app/helpers/data_getter.py

```diff
@@ -98,7 +98,7 @@ def get_invite_by_user_id(user_id):
     @staticmethod
     def get_all_events():
         """Method return all events"""
-        return Event.query.order_by(desc(Event.id)).filter_by(in_trash=False).all()
+        return Event.query.order_by(Event.timezone).filter_by(in_trash=False).all()
```

Comment:
How does ordering by Event.timezone make new events to come on top?? Can you please explain?